core/templates/dev/head/pages/skill_editor/editor_tab/skill_concept_card_editor_directive.html

```diff
@@ -1,109 +1,107 @@
-<div>
-  <md-card class="oppia-editor-card-with-avatar">
-    <div class="oppia-editor-card-body">
-      <div class="oppia-editor-card-section"
-           style="padding-bottom: 36px;">
-        <div>
-          <strong style="font-size: 1.4em;">Concept Card</strong>
-        </div>
+<md-card class="oppia-editor-card-with-avatar">
+  <div class="oppia-editor-card-body">
+    <div class="oppia-editor-card-section"
+         style="padding-bottom: 36px;">
+      <div>
+        <strong style="font-size: 1.4em;">Concept Card</strong>
```

Comment:
Try not to use inline styles, use classes instead. Ditto elsewhere.

Reply:
Done.